server/views/pages/cookiePolicy.njk

```diff
@@ -0,0 +1,106 @@
+
+      <h2 class="govuk-heading-m">Essential cookies</h2>
+      <p class="govuk-body">
+        Essential cookies keep your information secure while you use Refer and monitor an intervention. We do not need to ask permission to use them.
+      </p>
+      {{ govukTable({
+          firstCellIsHeader: true,
+          head: [
+              {
+                  text: "Name",
+                  classes: 'app-custom-class'
+              },
+              {
+                  text: "Purpose",
+                  classes: 'app-custom-class'
+              },
+              {
+                  text: "Expires",
+                  classes: 'app-custom-class'
+              }
+          ],
+          rows: [
+              [
+                  { text: "connect.sid" },
+                  { text: "Keeps you logged into the service" },
+                  { text: "1 hour" }
+              ],
+              [
+                  { text: "acceptAnalyticsCookies" },
+                  { text: "Saves your analytics cookie consent setting" },
+                  { text: "1 year" }
+              ]
+          ]
+      }) }}
+
+      <h2 class="govuk-heading-m">Analytics cookies (optional)</h2>
+      <p class="govuk-body">
+          With your permission, we use Microsoft's Application Insights to collect data about how you use Refer and monitor an intervention. This information helps us to improve our service.
+        </p>
+      <p class="govuk-body">Microsoft is not allowed to use or share our analytics data with anyone.</p>
+      <p class="govuk-body">Application Insights stores anonymised information about:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>the pages you visit on Refer and monitor an intervention and how long you spend on them</li>
+        <li>the device and browser you are using to view the service</li>
+        <li>any errors you see while using the service</li>
+      </ul>
+
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <h2 class="govuk-heading-l">Change your cookie settings</h2>
+          <form action="/cookie-preferences" method="post" novalidate>
+            <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+
+            {{ govukRadios({
+                idPrefix: "analytics-cookies",
+                name: "acceptAnalyticsCookies",
+                fieldset: {
+                    legend: {
+                        text: "Do you want to accept analytics cookies?",
+                        classes: "govuk-fieldset__legend--s"
+                    }
+                },
+                items: [
+                    {
+                        value: "true",
+                        text: "Yes",
+                        checked: acceptAnalyticsCookies === 'true'
+                    },
+                    {
+                        value: "false",
+                        text: "No",
+                        checked: acceptAnalyticsCookies === 'false'
```

Comment:
I know this was my suggestion but pseudo-boolean strings seem a bit funny. I wonder if it's worth coercing them into actual booleans in the router? But then when passing it back and forth between server and client they might revert